Mergin/configuration_dialog.py

```diff
@@ -88,6 +88,12 @@ def writeSettings(self):
                 QgsApplication.messageLog().logMessage(f"Mergin plugin: {str(e)}")
                 mc = None
 
+        QgsExpressionContextUtils.setGlobalVariable('mergin_url', url)
+        if mc:
+            QgsExpressionContextUtils.setGlobalVariable('mergin_username', username)
+        else:
+            QgsExpressionContextUtils.removeGlobalVariable('mergin_username')
+
```

Comment:
What if I just open QGIS without going to configuration dialog - are these variables going to be set?